src/hooks/use-popular-categories-offers-data copy.tsx

```
@@ -0,0 +1,45 @@
+
+  useEffect(() => {
+    const fetchOffersData = async () => {
+      try {
+        const { data } = await axiosClient.get<OfferResponse>(
+          '/offers/popular',
+          { params: { limit } }
+        )
+
+        const results = data.data.reduce(
+          (acc, category) => ({
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)